ddtrace/llmobs/_llmobs.py

```diff
@@ -1307,27 +1315,37 @@ def submit_evaluation_for(
 
             if not isinstance(timestamp_ms, int) or timestamp_ms < 0:
                 error = "invalid_timestamp"
-                raise ValueError("timestamp_ms must be a non-negative integer. Evaluation metric data will not be sent")
+                raise ValueError("timestamp_ms must be a non-negative integer.")
 
             if not label:
                 error = "invalid_metric_label"
                 raise ValueError("label must be the specified name of the evaluation metric.")
 
-            metric_type = metric_type.lower()
-            if metric_type not in ("categorical", "score"):
+            if not metric_type or metric_type.lower() not in ("categorical", "numerical", "score", "boolean"):
```

Comment:
```suggestion
            if not metric_type or metric_type.lower() not in ("categorical", "score", "boolean"):
```
`submit_evaluation_for()` does not support `numerical` eval metrics. `submit_evaluation()` does however, which is why we want to raise error if someone uses a `numerical` metric type in `submit_evaluation_for()`, but only log a warning and silently convert to a score metric in `submit_evaluation()`. Does this make sense?